packages/babel-core/src/config/files/plugins.js

```diff
@@ -19,6 +19,18 @@ const OTHER_PLUGIN_ORG_RE = /^(@(?!babel\/)[^/]+\/)(?![^/]*babel-plugin(?:-|\/|$
 const OTHER_PRESET_ORG_RE = /^(@(?!babel\/)[^/]+\/)(?![^/]*babel-preset(?:-|\/|$)|[^/]+\/)/;
 const OTHER_ORG_DEFAULT_RE = /^(@(?!babel$)[^/]+)$/;
 
+const nodePaths =
+  typeof process.env.NODE_PATH !== "undefined"
+    ? process.env.NODE_PATH.split(
+      os
+        .type()
+        .toLowerCase()
+        .indexOf("win") === 0
+          ? ";"
+          : ":"
```

Comment:
You can use `.split(path.delimiter)` (https://nodejs.org/api/path.html#path_path_delimiter)

Reply:
Ah.

Reply:
Think I've also missed an import.